src/App.css

```diff
@@ -0,0 +1,15 @@
+.courses-container {
```

Comment:
ok, I'll take that into account but since I've made lot of changes in this regard in the Task 2, I would suggest to leave this comment unchanged and wait for the Task 2 submission. I created a lot of css files there